test/extensions/quic_listeners/quiche/active_quic_listener_test.cc

```diff
@@ -111,6 +112,17 @@ class ActiveQuicListenerTest : public QuicMultiVersionTest {
     ON_CALL(listener_config_, listenSocketFactory()).WillByDefault(ReturnRef(socket_factory_));
     ON_CALL(socket_factory_, getListenSocket()).WillByDefault(Return(listen_socket_));
 
+    // Use UdpDefaultWriter to perform non-batched writes for the purpose of this test
+    ON_CALL(listener_config_, udpPacketWriterFactory())
+        .WillByDefault(Return(&udp_packet_writer_factory_));
+    ON_CALL(udp_packet_writer_factory_, createUdpPacketWriter(_, _))
+        .WillByDefault(Invoke(
+            [&](Network::IoHandle& io_handle, Stats::Scope& scope) -> Network::UdpPacketWriterPtr {
+              Network::UdpPacketWriterPtr udp_packet_writer =
+                  std::make_unique<Network::UdpDefaultWriter>(io_handle, scope);
```

Comment:
Can we use the gso writer for this test?

Reply:
Using Gso Writer For the test here now.